### PR TITLE
Sync various sections

### PR DESCRIPTION
**Pull request by saundefined**

### CODE REVIEW

File: reference/datetime/dateinterval/construct.xml [hunk 1]
```diff
@@ -141,6 +139,14 @@
      </row>
     </thead>
     <tbody>
+     <row>
+      <entry>8.2.0</entry>
+      <entry>
+       Seules les propriétés <literal>y</literal> à <literal>f</literal>,
+       <literal>inverser</literal>, et <literal>days</literal> seront visibles,
```
Comment (devnexen) [inline reference/datetime/dateinterval/construct.xml:146]: `invert` is not to translate I think.

Comment (saundefined) [reply reference/datetime/dateinterval/construct.xml]: Exactly! Thank you

File: language-snippets.ent [hunk 2]
```diff
@@ -2882,6 +2884,22 @@ vous devriez utiliser l&#39;extension <link linkend="set.mongodb">MongoDB</link>
...
 </row>
 '>
+
+<!ENTITY mongodb.option.let '
+ <row xmlns="http://docbook.org/ns/docbook">
+  <entry>let</entry>
+  <entry><type class="union"><type>array</type><type>object</type></type></entry>
+  <entry>
+   <para>
+    Carte des noms et des valeurs des paramètres. Les valeurs doivent être des constantes ou des expressions fermées qui ne font pas référence aux champs du document. Les paramètres peuvent ensuite être accédés en tant que variables dans un contexte d&#39;expression agrégée (par exemple <literal>$$var</literal>).
```
Comment (Girgias) [inline language-snippets.ent:2894]:
```suggestion
    Dictionnaire des noms et des valeurs des paramètres. Les valeurs doivent être des constantes ou des expressions fermées qui ne font pas référence aux champs du document. Les paramètres peuvent ensuite être accédés en tant que variables dans un contexte d&#39;expression agrégée (par exemple <literal>$$var</literal>).
```
As we are refrering to the data structure here ``tableau associatif`` would also work

File: reference/datetime/functions/time.xml [hunk 3]
```diff
@@ -20,6 +20,13 @@
    en secondes depuis le début de l'époque UNIX, (1er
    janvier 1970 00:00:00 GMT).
   </para>
+  <note>
+   <para>
+    Les timestamps Unix ne contiennent aucune information concernant le fuseau horaire local.
```
Comment (Girgias) [inline reference/datetime/functions/time.xml:25]:
```suggestion
    Les horodatages Unix ne contiennent aucune information concernant le fuseau horaire local.
```

**Comment by Girgias** [comment]
Thank you!

**Pull request merged**